src/components/AllCitiesOverviewContainer/AllCitiesOverview/AllCitiesOverview.scss

```diff
@@ -123,36 +123,57 @@ $round-image-width: 220px;
   font-weight: 500;
 }
 
+.subInfoWrapper {
+  @include shadow-inset-bottom;
+
+  background-color: $color-primary--020;
+}
+
 .subInfoContainer {
   @include container;
 
-  display: flex;
-  flex-direction: column;
   padding-top: 42px;
   padding-bottom: 42px;
+}
 
-  @include for-tablet-portrait-up {
-    flex-direction: row;
-  }
+.legend {
+  margin: 0 0 12px;
+  font-size: 22px;
+  font-weight: 300;
 }
 
-.subInfoTextWrapper {
-  flex: auto;
+.legendRow {
   margin-bottom: 32px;
 
-  @include for-tablet-portrait-up {
+  &:last-child {
     margin-bottom: 0;
-    margin-right: 64px;
+  }
+
+  @include for-tablet-landscape-up {
+    display: flex;
+    margin-bottom: 10px;
+  }
+}
+
+.icon {
+  margin-right: 14px;
+}
+
+.term {
+  font-weight: 500;
+  margin-bottom: 10px;
+
+  @include for-tablet-landscape-up {
+    flex-shrink: 0;
+    margin: 0 10px 0 0;
   }
 }
 
-.subInfoIcon {
-  margin: 0 auto;
+.definition {
+  margin: 0;
 
-  @include for-tablet-portrait-up {
-    flex: none;
-    width: 140px;
-    margin: 14px 0 0; // visually align the top with the title text
+  @include for-tablet-landscape-up {
+    flex-grow: 1;
```

Comment:
This isn't needed, is it? This text will just expand to take up whatever space it needs. If it is needed, `flex: auto` should be the preferred method for saying 'take all available space'.